### PR TITLE
Add malformed ELF files to bpf2c corpus

### PR DESCRIPTION
**Pull request by Alan-Jowett**
Resolves: #3114

## Description

Add malformed ELF files to bpfc2 corpus

## Testing

CI/CD

## Documentation

No.

## Installation

Yes - Removes unused malformed ELF files from installer.

### CODE REVIEW

File: tests/libfuzzer/bpf2c_fuzzer/bpf2c_fuzzer.vcxproj.filters [hunk 1]
```diff
@@ -45,4 +48,15 @@
...
     </CustomBuild>
   </ItemGroup>
+  <ItemGroup>
+    <Object Include="..\..\sample\bad\bad1.o">
+      <Filter>Malformed Object Files</Filter>
+    </Object>
+    <Object Include="..\..\sample\bad\bad2.o">
+      <Filter>Malformed Object Files</Filter>
+    </Object>
+    <Object Include="..\..\sample\bad\bad3.o">
```
Comment (dthaler) [inline tests/libfuzzer/bpf2c_fuzzer/bpf2c_fuzzer.vcxproj.filters:58]: Why is bad4.o not in this list?

**Review by dthaler — DISMISSED** [review]
Please fix typo in PR title (bpfc2 -> bpf2c)

File: tests/libfuzzer/bpf2c_fuzzer/bpf2c_fuzzer.vcxproj.filters (hunk 1 above)
Comment (Alan-Jowett) [reply tests/libfuzzer/bpf2c_fuzzer/bpf2c_fuzzer.vcxproj.filters]: Files have been renamed to a uniquely identify each malformed file and additional regression tests have been to verify that bpf2c and verifier are able to correctly detect and reject these files.

**Comment by dthaler** [comment]
PR description says Installation "No", but then the PR updates an installation file.

**Comment by Alan-Jowett** [comment]
Switching to draft until https://github.com/serge1/ELFIO/pull/132 reaches this repo.

**Comment by Alan-Jowett** [comment]
> Switching to draft until [serge1/ELFIO#132](https://github.com/serge1/ELFIO/pull/132) reaches this repo.
Rebased to include this fix.

**Review by dthaler — APPROVED** [review]

**Review by dv-msft — APPROVED** [review]

**Pull request merged**